tensorflow/core/platform/windows/env.cc

```diff
@@ -52,7 +52,16 @@ class StdThread : public Thread {
 
 class WindowsEnv : public Env {
  public:
-  WindowsEnv() {}
+  WindowsEnv()
+      : GetSystemTimePreciseAsFileTime_(NULL) {
+    HMODULE module = GetModuleHandle("kernel32.dll");
```

Comment:
Is it possible to add `kernel32` as one of the `target_link_libraries` and resolve this symbol statically? (I assume there is some reason for resolving it dynamically, so probably just ignorance on my part....)